modules/host/default.nix

```diff
@@ -16,4 +16,7 @@
 
   networking.hostName = "ghaf-host";
   system.stateVersion = "22.11";
+
+  # PCI passthrough needs larger locked-in-memory space than default
+  systemd.services."microvm@".serviceConfig.LimitMEMLOCK = 999999999;
```

Comment:
This line breaks the build

Reply:
This was already fixed in microvm.nix https://github.com/astro/microvm.nix/commit/68a02424760717fdab78ddf4cddf3eda6ffc6da4

Reply:
Ok, I'll wait for @panufi 's comment on `allow_unsafe_interrupts` before testing removing it.